### DISCUSSION TITLE
Оказывается эта волшебная штука работает на oculus quest 2 :^)

### DISCUSSION THREAD

**Discussion by yyebak** [dovecoteescapee/ByeDPIAndroid#195] [General]
Просто захотелось дать инфу для тех, кто ищет способы смотреть ютуб на квесте или иметь доступ к магазину и обновлениям. Мне пришлось потанцевать с бубном в side quest (наперед скажу, что там нужно создать свою организацию, но кому надо тот разберется), потом понять что на окулусе отдельны фильтр приложений поставленных извне. В общем, работает :D и все ради видосов в 360 без скачивания и смс........

**Comment by RomanKoff** [comment]
> потом понять что на окулусе отдельны фильтр приложений поставленных извне.

Можно более развернутый намек, где копать? apk поставил и commandline прописал, но не работает...

**Reply by yyebak** [reply]
кароче, насколько я помню, зайти в список приложений и там есть фильтр выпадающее окошко такое и там неизвестные источники, оттуда запускается байдпи, но вы, наверное, уже разобрались
дальше, надо в режиме разработчика в настройках самого квеста включить мультизадачность (multitask или чет такое) чтобы можно было держать 3 открытых приложения... потом как я помню я себе ставила в настройках байдпи автозапуск и оно просто работало в фоне постоянно и все грузилось
если будут вопросы еще то спрашивайте помогу чем смогу
пс. а дальше уже танец с бубном от того какая команда сработает а какая нет

**Reply by RomanKoff** [reply]
Все понял... Я не на то подумал) Все работает. Спасибо.

**Reply by MrLogiousBanana** [reply]
Псст.. Слушайте, а можно как-нибудь настроить DPI'ку, чтобы не только мета-сервисы работали, но и игры запускались? Лично у меня и VR Chat и VAIL в нативе жалуются, что подключения нет, ни смотря на то, что DPI активен. Может надо прописать ручками какой адрес?

**Reply by RomanKoff** [reply]
ByDPI не универсальное лекарство. Иногда там нужна тонкая настройка параметров со знанием дела, а иногда и она не поможет. Другое решение это vpn, если bydpi не смогло...

**Reply by Herly1X** [reply]
А на квест 3s byedpi можно скачать?

**Reply by Visor81** [reply]
да, ставится без проблем. Ютуб как часы работает
